GeoPoint[] waypoints
---
bool success
string message